packages/block-editor/src/components/index.js

```diff
@@ -104,6 +104,9 @@ export {
  */
 
 export { default as __unstableBlockSettingsMenuFirstItem } from './block-settings-menu/block-settings-menu-first-item';
+export { default as __unstableCommentIconFill } from './collab/block-comment-icon-slot';
```

Comment:
This should be a private API. Examples: packages/block-editor/src/private-apis.js
We wouldn't want the merge of the experiment to add this public APIs.